secator/tasks/cariddi.py

```
@@ -132,35 +133,44 @@ def on_json_loaded(self, item):
-				)
+				extra_data = {k: v for k, v in param.items() if k not in ['name', 'attacks']}
+				extra_data['content'] = attack
+				parsed_url = urlparse(url)
+				params = parsed_url.query.split('&')
+				url_without_param = urlunparse(parsed_url._replace(query=''))
+				for p in params:
+					p_name, p_value = p.split('=')
+					if p_name == param_name:
+						p_value = p_value
+						break
+					yield Tag(name=p_name, category='url_param', match=url_without_param, extra_data={'content': p_value, 'subtype': 'param'})  # noqa: E501
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)